website/static/js/registrationUtils.js

```diff
@@ -210,6 +210,9 @@ var Question = function(questionSchema, data) {
         value = self.data.value();
     } else {
         value = self.data.value || null;
+        var elem = document.createElement('textarea');
```

Comment:
Can you write a JS test that tries to inject a script and makes sure it doesn't work?

Reply:
@brianjgeiger If this work as expected, is there any reason we couldn't replace the other amp cleaners and such with this method? E.g., decodeText in osfhelpers

Reply:
Maybe. It would be worth exploring, for sure. I don't know if there are any other consequences to the field, but it would probably be fine.

Reply:
I'll give it a go, might take a while. I searched for a current test of registration form data, and couldn't find one.

Reply:
I'm also, especially if we want to generalize this, open to making this a separate function that you can test independently. That's probably the easier way to go.

Reply:
I've added textareaElementDecode to osfHelpers as a separate function. However, I have had no luck trying to test for HTML injection. The biggest problem is I have been unable to create a negative result (injection of any kind). However, even if I were to finally contrive an injection, it could only be tested after running though the mako template. Not sure how that would be done.